dev_shutdown.groovy

```diff
@@ -0,0 +1,122 @@
+// using declarative pipeline: https://jenkins.io/doc/book/pipeline/syntax/
+@Library('PipelineUtils@master')
+import dvsa.aws.mot.jenkins.pipeline.common.AWSFunctions
+import dvsa.aws.mot.jenkins.pipeline.common.RepoFunctions
+import dvsa.aws.mot.jenkins.pipeline.common.GlobalValues
+
+// global variables
+AWSFunctions awsFunctionsFactory = new AWSFunctions()
+RepoFunctions repoFunctionsFactory = new RepoFunctions()
+GlobalValues globalValuesFactory = new GlobalValues()
+Map<String, Map<String, String>> gitlab = globalValuesFactory.GITLAB_REPOS()
+
+Map<String, String> params = [
+  exclusions_raw : "${EXCLUDED_ENVS}"
+]
+
+List<String> envs_list      = []
+List<String> exclusions     = []
+List<String> lull_list      = []
+String project              = 'cvr'
+String tf_branch            = 'master'
+String jenkinsCtrlNodeLabel = "ctrl"
+String account              = "dvsarecallsdev"
+def all_envs_raw
+def account_id
+
+// PIPELINE --------------------------------------------------------------------------
+pipeline {
+  agent none
+  options {
+    ansiColor('xterm')
+    timestamps()
+    timeout(time: 1, unit: 'HOURS')
+    buildDiscarder(
+        logRotator(
+            daysToKeepStr: '7'
+        )
+    )
+  }
+  stages {
+    stage ('Init') {
```

Comment:
Not sure whether `Init` is a right name for the entire job. I'd say we don't even need a wrapper stage for those two stages